### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lawteo**
Updated a typo in the example for the header option parameter.

### CODE REVIEW

**Comment by bvschwartz** [comment]
Please accept this!

**Comment by Qiming-Liu** [comment]
In javascript, those two codes are equivalent.

**Comment by lawteo** [comment]
> In javascript, those two codes are equivalent.

Not really, one is a label & value, the other is just the value.
Testing shows that without the label, it is not working.

**Comment by Qiming-Liu** [comment]
> Not really, one is a label & value, the other is just the value. Testing shows that without the label, it is not working.

When declaring an object, the two ways of writing are indeed the same. 

image

Maybe in this case, it is the function parameters will be different. (I'm not sure about this)
Don't know why the testing not working.

**Comment by lawteo** [comment]
> > Not really, one is a label & value, the other is just the value. Testing shows that without the label, it is not working.
> 
> When declaring an object, the two ways of writing are indeed the same.
> 
> image
> 
> Maybe in this case, it is the function parameters will be different. (I'm not sure about this) Don't know why the testing not working.

In your code, you first defined:
const header = 'header';
that is why it is working as you say.
if you did not do that, it won't be like what you are showing.
In other words, the coder must declare a variable named header with the desired value. (which the coder is not aware)
Else the code will not work.

In my suggested code, we are doing that for the coder so they do not need to.

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ const csvFromArrayOfObjects = convertArrayToCSV(dataObjects);
   const csvFromArrayOfArrays  = 'number;first;last;handle\n1;Mark;Otto;@mdo\n2;Jacob;Thornton;@fat\n3;Larry;the Bird;@twitter\n';
 */
 const csvFromArrayOfArrays = convertArrayToCSV(dataArrays, {
-  header,
+  header: header,
```
Comment (aichbauer) [inline README.md:103]: On L67 in this file we actually declare the header `const header = ['number', 'first', 'last', 'handle'];` so in this case the example would work just fine.

See the previous comments of @Qiming-Liu for more information.

**Pull request closed without merging**